src/components/ArticleComments.jsx

```diff
@@ -24,9 +24,9 @@ export default function ArticleComments(props) {
         })
     }, [])
 
-    function getCommentAuthor (users, comment) {
+    function getUserData (users, content) {
```

Comment:
this functionality is not really relevant to this ticket and isn't so useful right now - Try not to add any additional code that isn't required to complete the task that this branch is referring to - so that we can keep all relevant code on the relevant branch

Reply:
Sorry Saima - I thought it was relevant as I needed it to get the commenter's profile pic. But I appreciaet this was beyond the scope of the ticket. Ta! 👍